doc/pages/tutorials/010-astarte_in_5_minutes.md

```diff
@@ -60,15 +60,17 @@ $ astartectl housekeeping realms ls --housekeeping-url http://localhost:4001/ -k
 
 ## Install an interface
 
-We will use [Astarte's Qt5 Stream Generator](https://github.com/astarte-platform/stream-qt5-test) to feed data into Astarte. Clone the repository, as we will have to install its `org.astarteplatform.Values` interface into our new realm. To do that, we can use `astartectl` again:
+We will use Astarte’s standard interfaces for our test. You can install them by running:
 
-```sh
-$ astartectl realm-management interfaces install ../stream-qt5-test/interfaces/org.astarteplatform.Values.json --realm-management-url http://localhost:4000/ -r test -k test_private.pem
+```
+$ astartectl realm-management interfaces install standard-interfaces/org.astarte-platform.genericsensors.Values.json --realm-management-url http://localhost:4000/ -r test -k test_private.pem
```

Comment:
Instead of these, we should use the new astartectl sync command. So this would become:

`astartectl realm-management interfaces sync standard-interfaces/*.json --realm-management-url http://localhost:4000/ -r test -k test_private.pem`